src/java_bytecode/java_types.cpp

```diff
@@ -286,8 +291,8 @@ std::string gather_full_class_name(const std::string &src)
 
 /// Given a substring of a descriptor or signature that contains one or more
 /// types parse out the individual types. This is used for parsing the
-/// parameters of a function or the generic arguments contained within angle
-/// brackets.
+/// parameters of a function or the generic type variables contained within
+/// angle brackets.
```

Comment:
might make sense to specify that it will also parse method signatures which are wrapped in `(`, `)` brackets

Reply:
The beginning of the sentence says 'This is used for parsing the parameters of a function or...', would it be enough to change it to 'This is used for parsing the parameters of a function contained within parentheses or..' or did you mean something else?